skunkworks_crow/src/main/res/layout/activity_main.xml

```diff
@@ -21,37 +21,65 @@
             android:layout_centerHorizontal="true"
             android:layout_centerVertical="true"
             android:gravity="center"
-            android:textSize="22sp"
-            android:padding="10dp"/>
+            android:padding="10dp"
+            android:textSize="22sp" />
 
         <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/recyclerview"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:id="@+id/recyclerview"/>
+            android:layout_height="match_parent" />
 
     </RelativeLayout>
 
     <LinearLayout
         android:id="@+id/buttonholder"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="?android:actionBarSize"
         android:layout_alignParentBottom="true"
-        android:orientation="horizontal"
-        android:weightSum="2">
+        android:background="#ffffff"
```

Comment:
Please avoid using raw values wherever possible.
This can be replaced with `@android:color/white`

Reply:
Done instead of `@android: color/white` I used `@android: color/colorTabActive` as this is initialized before only

Reply:
@Chromicle Sometimes variable name or resources name may confuse someone(as in this case colorTabActive is reserved for tabs related activity but if someone in future changes this value then it will reflect here as well) so its better to use `@android:color/white`

Reply:
> @Chromicle Sometimes variable name or resources name may confuse someone(as in this case colorTabActive is reserved for tabs related activity but if someone in future changes this value then it will reflect here as well) so it's better to use `@android:color/white`

Done